### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by apple1417** [apple1417/bl2.parts#9]
This is basically an open call for developers, I'm not a web dev and am not really familiar with how to do this properly. There are two obvious issues even I can see as an amateur:
- There is a lot of information which is only communicated via colour
- People semi-frequently have trouble telling between "+1" and "÷1" in the bonuses (I certainly do without my glasses)

On top of that, I suspect there are likely issues with the total lack of aria tags, and that the layout of each part stat box might cause issues with screen readers.

Rough requirements:
- Make it better.
- Mostly keep the current page layout *visually*. If the actual HTML elements need to be rearranged to flow better, so be it.
- No pulling in large external dependencies. I'm ok pulling in dependencies that do one specific thing well (e.g. anchor.js), but I don't want any large frameworks.
- Minimize javascript, the site should mostly work under noscript. For example, all the formula pages fall back to plain text if MathJax is blocked. It might mean the site works less effectively, but it shouldn't completely break it.
- No LLMs

Please consult with me before putting any significant effort in.
